community/lilo/build

```diff
@@ -20,7 +20,7 @@ sed -i "/images all/d"                     Makefile src/Makefile
 
 export LC_ALL=C
 
-make CFLAGS="$CFLAGS -fcommon" all
+make CFLAGS="-fcommon" all
```

Comment:
Do we want to clear the user's CFLAGS?

Reply:
Gentoo explicitly removes them for stability reasons.